clients/cobol-lsp-vscode-extension/src/extension.ts

```diff
@@ -13,9 +13,17 @@
  */
 
 import * as vscode from "vscode";
-import { __ExtensionApi } from "@code4z/cobol-dialect-api";
-import { isV1RuntimeDialectDetail } from "./dialect/utils";
 import { gotoCopybookSettings } from "./commands/OpenSettingsCommand";
+import {
+  __ExtensionApi,
+  CopyStatementParser,
+  V2DialectDetail,
+} from "@code4z/cobol-dialect-api";
+import { V2StartProcessingHandler } from "@code4z/cobol-dialect-api";
```

Comment:
Can all the imports of `@code4z/cobol-dialect-api` in the main extension be just type imports?
```suggestion
import type {
  __ExtensionApi,
  CopyStatementParser,
  V2DialectDetail,
} from "@code4z/cobol-dialect-api";
import type { V2StartProcessingHandler } from "@code4z/cobol-dialect-api";
```